### PR TITLE
Tool-4 Create sql script

### PR DESCRIPTION
**Pull request by ani-hyan**

### CODE REVIEW

File: movie_flow_create.sql [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+    name TEXT NOT NULL,
+    founded_date DATE
+);
+
+CREATE TABLE IF NOT EXISTS rating (
+	id SERIAL PRIMARY KEY,
+    rate DOUBLE PRECISION NOT NULL,
+    max_rate DOUBLE PRECISION NOT NULL,
+    movie_id INT NOT NULL,
+    FOREIGN KEY (movie_id) REFERENCES movie(id) ON UPDATE NO ACTION ON DELETE NO ACTION NOT VALID
```
Comment (mchinarev) [inline movie_flow_create.sql]: Please use ON DELETE CASCADE ON UPDATE CASCADE for FKs + simplify:
```
movie_id INT NOT NULL REFERENCES movie(id) ON DELETE CASCADE ON UPDATE CASCADE
`

File: movie_flow_create.sql [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+);
+
+CREATE TABLE IF NOT EXISTS n2n_movie_to_actor (
+    movie_id INT NOT NULL,
+    actor_id INT NOT NULL,
+    PRIMARY KEY (movie_id, actor_id),
+    FOREIGN KEY (movie_id) REFERENCES movie(id) ON UPDATE NO ACTION ON DELETE NO ACTION NOT VALID,
+    FOREIGN KEY (actor_id) REFERENCES actor(id) ON UPDATE NO ACTION ON DELETE NO ACTION NOT VALID
+);
```
Comment (mchinarev) [inline movie_flow_create.sql]:
```suggestion
CREATE TABLE IF NOT EXISTS n2n_movie_to_actor (
    movie_id INT NOT NULL REFERENCES movie(id) ON DELETE CASCADE ON UPDATE CASCADE,
    actor_id INT NOT NULL REFERENCES actor(id) ON DELETE CASCADE ON UPDATE CASCADE
);
```

File: movie_flow_create.sql [hunk 3]
```diff
@@ -0,0 +1,77 @@
+CREATE SCHEMA IF NOT EXISTS movie_flow;
+SET search_path = movie_flow;
+
+CREATE TABLE IF NOT EXISTS movie (
+    id SERIAL PRIMARY KEY,
+    release_date TIMESTAMP NOT NULL,
```
Comment (mchinarev) [inline movie_flow_create.sql]: You have TIMESTAMP datatype here but in data insertion script you are inserting values like '2022-01-15', '2019-05-02' without specified time, so looks like we can change type to just DATE

File: movie_flow_data.sql [hunk 4]
```diff
@@ -0,0 +1,153 @@
...
+    ('Rotten Tomatoes', 8.8, 10, 5),  -- Mad Max: Fury Road
+    ('IMDb', 8.2, 10, 6),  -- A Star Is Born
+    ('Rotten Tomatoes', 9.5, 10, 7),  -- Knives Out
+    ('Metacritic', 8.4, 10, 8),  -- Zootopia
+    ('IMDb', 8.6, 10, 9),  -- Interstellar
+    ('Rotten Tomatoes', 7.7, 10, 10);  -- The Heat
+
+INSERT INTO n2n_movie_to_actor (movie_id, actor_id)
+VALUES 
+    (1, 1),  -- Inception - Leonardo DiCaprio
```
Comment (mchinarev) [inline movie_flow_data.sql:94]: Nice to see that you commented it, this is a good practice while working with IDs 👍

File: movie_flow_data.sql [hunk 5]
```diff
@@ -0,0 +1,153 @@
...
+    (6, 13),  -- A Star Is Born - Bradley Cooper
+    (7, 14),  -- Knives Out - Ana de Armas
+    (8, 15),  -- Zootopia - Chris Evans
+    (9, 16),  -- Interstellar - Ginnifer Goodwin
+    (10, 17),  -- The Heat - Jason Bateman
+    (9, 18),  -- Interstellar - Matthew McConaughey
+    (10, 19),  -- The Heat - Anne Hathaway
+    (10, 20);  -- The Heat - Melissa McCarthy
+
+INSERT INTO n2n_movie_to_external_platform (movie_id, external_platform_id, movie_url)
```
Comment (mchinarev) [inline movie_flow_data.sql]: we also need to rename here :)

Comment (ani-hyan) [reply movie_flow_data.sql]: done!

**Review by mchinarev — APPROVED** [review]

**Pull request merged**